CPAC/distortion_correction/utils.py

```diff
@@ -0,0 +1,339 @@
+import os
+import sys
+import subprocess
+import numpy as np
+import nipype
+import nibabel
+from nipype.pipeline import engine as pe
+from nipype.interfaces import fsl
+from nipype.interfaces import utility
+import nipype.interfaces.utility as util
+from nipype.interfaces.fsl import utils as fsl_utils
+from nipype.interfaces.fsl import maths as fsl_maths
+from nipype.interfaces.fsl import ExtractROI
+from nipype.interfaces.fsl import maths
+
+
+def run_HCP_gradient_unwarp(phase_vol, input_coeffs):
+
+    import os
+    import subprocess
+
+    trilinear = os.path.join(os.getcwd(), "trilinear.nii.gz")
+
+    cmd = [
+        "/home/tgeorge/gradunwarp/gradunwarp/core/gradient_unwarp.py",
```

Comment:
Is this [this](https://github.com/Washington-University/gradunwarp/blob/v1.2.0/gradunwarp/core/gradient_unwarp.py)?

We should add a dependency (either to C-PAC itself or to the Dockerfile) instead of relying on a local path here

Reply:
It is, and we're holding off for now (apparently, it is not commonly used, and it is optional). I'll remove this path though.